tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "esModuleInterop": true,
     "strict": true,
     "moduleResolution": "node",
+    "skipLibCheck": true,
```

Comment:
was needed because: 

```

ERROR in node_modules/react-chartist/dist/index.d.ts(1,10):
TS2440: Import declaration conflicts with local declaration of 'ChartitGraphProps'.

```